src/Tests/VideoBundleTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Drupal\media\Tests;
+
+use Drupal\simpletest\WebTestBase;
+
+/**
+ * Ensures that media bundle for videos can be created.
+ *
+ * @group media
+ */
+class VideoBundleTest extends WebTestBase {
+
+    /**
+     * Modules to enable.
+     *
+     * @var array
+     */
+    public static $modules = ['media', 'media_entity', 'video_embed_field', 'video_embed_media'];
+
+    /**
+     * The test media bundle.
+     *
+     * @var \Drupal\media_entity\MediaBundleInterface
+     */
+    protected $testBundle;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp() {
+        parent::setUp();
+        $this->testBundle = $this->container->get('entity.manager')->getStorage('media_bundle')->load('video');
+    }
+
+    /**
+     * Tests video media bundle creation from config files.
+     */
+    public function testMediaBundleCreationFromModule() {
+        $type_configuration = [
+          'source_field' => 'field_video'
+        ];
+
+        $field_map = [
+          'id' => 'field_id',
+          'source_name' => 'field_source'
+        ];
+
+        $this->assertTrue((bool) $this->testBundle, 'The media bundle from default configuration has been created in the database.');
+        $this->assertEqual($this->testBundle->get('label'), 'Video', 'Correct label detected.');
+        $this->assertEqual($this->testBundle->get('description'), 'Use Video for embedding videos hosted by YouTube, Vimeo, or some other provider.', 'Correct description detected.');
+        $this->assertEqual($this->testBundle->get('type'), 'video_embed_field', 'Correct plugin ID detected.');
+        $this->assertEqual($this->testBundle->get('type_configuration'), $type_configuration, 'Type configuration correct.');
+        $this->assertEqual($this->testBundle->get('field_map'), $field_map, 'Correct field map detected.');
```

Comment:
Would be nice to have another test that creates entity and checks if this fields were actually populated.